packages/mercury/src/components/paginator/_paginator-styles.scss

```
@@ -0,0 +1,271 @@
+@use "../../../../common/resets" as *;
+@use "../../base/common" as *;
+@use "../../base/label/label-styles" as *;
+@use "../combo-box/combo-box-styles" as *;
+@use "../input/input-styles" as *;
+
+%paginator {
+  @extend %body-regular-s;
+  display: grid;
+  gap: var(--mer-spacing--xs);
+  grid-template: "items-per-page items-per-page-info . first__button prev__button go-to next__button last__button" 1fr / auto auto 1fr auto auto auto auto;
+
+  &__combo-box {
+    border: 1px solid var(---mer-border-color__primary--focused, #d2ddeb);
```

```suggestion
    border: var(--mer-border__width--sm) solid var(---mer-border-color__primary--focused);
```

- Whenever possible use Mercury tokens instead of fixed values.
- There is no need for a fallback border-color value. If you use fallbacks, use Mercury variables as well.